packs/rollable-tables/11th-level-permanent-items.json

```diff
@@ -63,7 +63,7 @@
                 24
             ],
             "text": "Bag of Holding (Type III)",
```

Comment:
This is a spacious pouch now right?

Reply:
Yeah, I only fixed the 10th and 1st level table links